apps/examples/testcase/le_tc/kernel/tc_signal.c

```diff
@@ -181,7 +185,7 @@ static void tc_signal_sigaction(void)
 	TC_ASSERT_EQ("sigaction", st_act.sa_handler, sigaction_handler);
 
 	/* make sure action is not changed */
-	sigact_after = sig_findaction(sched_self(), SIGINT);
+	sigact_after = (FAR sigactq_t *)ioctl(fd, TESTIOC_GET_SIG_FINDACTION_ADD, 0);
```

Comment:
I think it needs to check whether both sigact_after and sigact_before are error or not

Reply:
good. thanks.